plugins/feedback-backend/src/api/jiraApiService.ts

```diff
@@ -0,0 +1,106 @@
+import axios from 'axios';
+
+export const createJiraTicket = async (options: {
+  host: string;
+  authToken: string;
+  projectKey: string;
+  summary: string;
+  description: string;
+  tag: string;
+  feedbackType: string;
+  reporter?: string;
+}): Promise<any> => {
+  const {
+    host,
+    authToken,
+    projectKey,
+    summary,
+    description,
+    tag,
+    feedbackType,
+    reporter,
+  } = options;
+  let body: any = {
+    fields: {
+      project: {
+        key: projectKey,
+      },
+      summary: summary,
+      description: description,
+      labels: ['reported-by-backstage', tag.toLowerCase().split(' ').join('-')],
+      issuetype: {
+        name: feedbackType === 'BUG' ? 'Bug' : 'Task',
+      },
+    },
+  };
+  if (reporter) {
+    body = {
+      fields: {
+        reporter: {
+          name: reporter,
+        },
+        project: {
+          key: projectKey,
+        },
+        summary: summary,
+        description: description,
+        labels: [
+          'reported-by-backstage',
+          tag.toLowerCase().split(' ').join('-'),
+        ],
+        issuetype: {
+          name: feedbackType === 'BUG' ? 'Bug' : 'Task',
+        },
+      },
+    };
+  }
```

Comment:
```suggestion
 const body = {
      fields: {
        ...(reporter && {reporter: {
          name: reporter,
        }}),
        project: {
          key: projectKey,
        },
        summary: summary,
        description: description,
        labels: [
          'reported-by-backstage',
          tag.toLowerCase().split(' ').join('-'),
        ],
        issuetype: {
          name: feedbackType === 'BUG' ? 'Bug' : 'Task',
        },
      },
    };
  }
```

Reply:
I have removed the duplicated code and updated the `package.json` in latest commit